source/services/encryption/EncryptionService.ts

```diff
@@ -56,15 +61,39 @@ export async function encryptWithAesKey(
   );
 }
 
+export async function encryptWithSymmetricKey(
+  forms: AnsweredForm,
+  plaintextAnswers: string
+): Promise<string> {
+  const symmetricKey = await getStoredSymmetricKey(forms);
+  const encryptedAnswers = await Aes.encrypt(
+    plaintextAnswers,
+    symmetricKey.toString(16).padStart(64, "0"),
+    "003d8999f6a4bb9800ed24b5d1846523"
```

Comment:
This IV magic string should probably be a constant since it's reused in the decrypt function

Reply:
Moved the IV and other magic values into a separate object 🙂.